### PR TITLE
[neato] Updated the readme with the correct channel configuration for items

### PR DESCRIPTION
**Pull request by yveshanoulle**
The readme is now updated with the configuration needed to make the items work.
A user can now copy past the example and it should work out of the box.

### CODE REVIEW

File: addons/binding/org.openhab.binding.neato/README.md [hunk 1]
```diff
@@ -68,20 +68,20 @@ Below you will find examples of the necessary files:
...
+    Switch FannDammIsDocked  "In dock [MAP(neato-sv.map):%s]" <present> (GNeato) { channel = "neato:vacuumcleaner:fanndamm:is-docked" }
+    Switch FannDammIsScheduled  "Scheduled [%s]" (GNeato) { channel = "neato:vacuumcleaner:fanndamm:is-scheduled" }
+    Switch FannDammIsCharging  "Is Charging [%s]" <heating> (GNeato) { channel = "neato:vacuumcleaner:fanndamm:is-charging" }
+    String FannDammCategory  "Cleaning Category [MAP(neato-sv.map):%s]" (GNeato)  { channel = "neato:vacuumcleaner:fanndamm:cleaning-category" }
+    String FannDammMode  "Cleaning Mode [MAP(neato-sv.map):%s]" (GNeato) { channel = "neato:vacuumcleaner:fanndamm:bcleaning-mode" }
+    String FannDammModifier  "Cleaning Modifier [MAP(neato-sv.map):%s]" (GNeato) { channel = "neato:vacuumcleaner:fanndamm:cleaning-modifier" }
+    Number FannDammSpotWidth  "SpotWidth [%.0f]" <niveau> (GNeato) { channel = "neato:vacuumcleaner:fanndamm:cleaning-spotwidth" }
+    Number FannDammSpotHeight  "SpotHeight [%.0f]" <niveau> (GNeato)  { channel = "neato:vacuumcleaner:fanndamm:cleaning-spotheigh" }
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.neato/README.md]: Missing a `t` at the end.
```suggestion
    Number FannDammSpotHeight  "SpotHeight [%.0f]" <niveau> (GNeato)  { channel = "neato:vacuumcleaner:fanndamm:cleaning-spotheight" }
```

File: addons/binding/org.openhab.binding.neato/README.md [hunk 2]
```diff
@@ -68,20 +68,20 @@ Below you will find examples of the necessary files:
...
+    String FannDammState  "Status [MAP(neato-sv.map):%s]" (GNeato) { channel = "neato:vacuumcleaner:fanndamm:state" }
+    String FannDammError  "Error [%s]" (GNeato) { channel = "neato:vacuumcleaner:fanndamm:error" }
+    String FannDammAction  "Action [MAP(neato-sv.map):%s]" (GNeato) { channel = "neato:vacuumcleaner:fanndamm:action" }
+    Switch FannDammDockHasBeenSeen  "Seen dock [%s]" <present> (GNeato) { channel = "neato:vacuumcleaner:fanndamm:dock-has-been-seen" }
+    Switch FannDammIsDocked  "In dock [MAP(neato-sv.map):%s]" <present> (GNeato) { channel = "neato:vacuumcleaner:fanndamm:is-docked" }
+    Switch FannDammIsScheduled  "Scheduled [%s]" (GNeato) { channel = "neato:vacuumcleaner:fanndamm:is-scheduled" }
+    Switch FannDammIsCharging  "Is Charging [%s]" <heating> (GNeato) { channel = "neato:vacuumcleaner:fanndamm:is-charging" }
+    String FannDammCategory  "Cleaning Category [MAP(neato-sv.map):%s]" (GNeato)  { channel = "neato:vacuumcleaner:fanndamm:cleaning-category" }
+    String FannDammMode  "Cleaning Mode [MAP(neato-sv.map):%s]" (GNeato) { channel = "neato:vacuumcleaner:fanndamm:bcleaning-mode" }
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.neato/README.md]: A `b` to much.

```suggestion
    String FannDammMode  "Cleaning Mode [MAP(neato-sv.map):%s]" (GNeato) { channel = "neato:vacuumcleaner:fanndamm:cleaning-mode" }
```

**Comment by cweitkamp** [comment]
> Do you think you can [sign-off](https://www.openhab.org/docs/developer/contributing/contributing.html#sign-your-work) your work.

@yveshanoulle Did you see my question about the sign-off?

**Comment by yveshanoulle** [comment]
> @yveshanoulle Did you see my question about the sign-off?
This is an update to a read me, no code, I did not see the point of signing. my ego is not so big to want my name linked to this.

**Pull request merged**